packages/react-components/react-tooltip/src/stories/TooltipPositioning.stories.tsx

```diff
@@ -38,7 +38,7 @@ export const Positioning = () => {
       }}
     >
       {positions.map(([position, icon]) => (
-        <Tooltip withArrow positioning={position} content={position} relationship="label">
+        <Tooltip key={`${position}${icon}`} withArrow positioning={position} content={position} relationship="label">
```

Comment:
_nit:_ I think it's a little overkill to use `icon` here since they are JSX and given that `position` is enough for the keys to be unique in this scenario.

```suggestion
        <Tooltip key={`${position}`} withArrow positioning={position} content={position} relationship="label">
```